src/views/Plant.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="plant-page">
+    <div class="header">
+      <BaseButton
+        width="160"
+        height="73"
+        color="#FFB800"
+        :active="buttonActive == 1"
+        radius="30"
+        @click="changeActive(1)"
+        >adottate</BaseButton
+      >
+      <BaseButton
+        width="160"
+        height="73"
+        color="#FFB800"
+        :active="buttonActive == 2"
+        radius="30"
+        @click="changeActive(2)"
+        >regalate</BaseButton
+      >
+    </div>
+    <div class="body">
```

Comment:
Come vedi questa é la parte importante del componente ed in effetti con un componente `TabBar` come ti dicevo sopra andresti a ridurre le linee di codice di molto rendendo questa div ( che é quella di maggiore interesse della pagina )`prevalente` nel componente.